docs/website/data/integrations.yaml

```diff
@@ -1,4 +1,18 @@
 integrations:
+  sphinx-rego:
+    title: Automatically document Rego policies
+    description: Sphinx extension that automatically documents Open Policy Agent Rego policies using meta properties.
+    software: 
+      - sphinx-doc
+     code:
+      - https://github.com/zenitysec/sphinx-rego
+    inventors:
+      - zenity
```

Comment:
You'll want to add this to the `organizations` list at the bottom of the document.